.github/workflows/coverage.yaml

```diff
@@ -4,6 +4,13 @@ on:
   push:
     branches: ["main"]
 
+env:
```

Comment:
To prevent config hell during our tests. Hardcode the `IP:HOST` and `USERNAME:PASSWORD@DATABASE` present in the `docker-compose.yaml` so anyone can run tests by running two command only without manually setting ENV in the compose.